code/src/components/products-container/ProductsContainer.test.tsx

```diff
@@ -64,6 +64,15 @@ const mockProducts: Product[] = [
   }
 ];
 
+jest.mock("@/hooks/use-snackbar/useSnackbar", () => ({
+  __esModule: true,
+  default: jest.fn(() => ({ openSnackbar: () => {} }))
```

Comment:
Without `() => ({ openSnackbar: () => {} })`. Test falls with `Cannot read properties of undefined (reading 'openSnackbar')`. I guess that is because inside returned by hook values is destructured.